integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/NtpSyncChecker.java

```diff
@@ -0,0 +1,42 @@
+package no.difi.meldingsutveksling;
+
+import lombok.extern.slf4j.Slf4j;
+import no.difi.meldingsutveksling.config.IntegrasjonspunktProperties;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.net.UnknownHostException;
+
+@Component
+@Slf4j
+public class NtpSyncChecker {
+
+    IntegrasjonspunktProperties props;
+    NTPClient client;
+
+    @Autowired
+    NtpSyncChecker(IntegrasjonspunktProperties props) throws UnknownHostException {
+        this.props = props;
+        client = new NTPClient(props.getNtpHost());
+    }
+
+    @Scheduled(fixedRate = 1000*60*30)
+    private void checkNtpOffset() {
+        log.debug("Checking NTP offset");
+        long offset = 0;
+        try {
+            offset = client.getOffset();
+        } catch (IOException e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1166)